wolox-books/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
```

Comment:
Add the env files to the gitignore please 😄

Reply:
Changed to example env files